cluster/olm/clusterserviceversion.yaml.tmpl

```diff
@@ -24,14 +24,155 @@ metadata:
     alm-examples: |-
       [
         {
-           "apiVersion": "pkg.crossplane.io/v1",
-           "kind": "Configuration",
-           "metadata": {
-              "name": "xp-getting-started-with-aws"
-           },
-           "spec": {
-              "package": "registry.upbound.io/xp/getting-started-with-aws:v1.3.0"
-           }
+          "apiVersion":"pkg.crossplane.io/v1",
+          "kind":"Configuration",
+          "metadata":{
+            "name":"xp-getting-started-with-aws"
+          },
+          "spec":{
+            "package":"registry.upbound.io/xp/getting-started-with-aws:v1.2.2"
```

Comment:
@hasheddan How do we keep these versions updated in new versions of Crossplane? Is there an automation or just manual?

Reply:
Just manually right now, but we could definitely look into getting this automated :)